docs/guides/docker_image_with_your_release.md

```diff
@@ -0,0 +1,58 @@
+# Build a docker image with your release
+
+Now that we have seen how to build a release, let's build one in a docker image. This is necessary for a deployment with docker swarm or kubernetes for example.
+
+A prerequisite step to this, is to initialize your release (`mix release.init`) and configure your [vm.args](/files/vm.args).
+
+once that is done add a `dockerfile` to your project with the following content
+
+```
+# check https://hub.docker.com/_/elixir/ for the appropriate version of elixir you want to run
+# alpine is used here to make dependency fetching faster
+FROM elixir:1.7.2-alpine
+
+# these arg command, set a default for the env variables
+ARG APP_NAME=your_app_name
+ARG MIX_ENV=prod
+ARG APP_VERSION=0.0.0
```

Comment:
I found this in some repo from the financial times, I thought that customizing version from when building the image was convenient. Let me know if you don't like it.